### PR TITLE
Travis CI new config

### LINKED ISSUE #85: Travis CI need update
**Description**:

Travis CI build failed in the recent PR

**Actual Behavior**:

Travis build failing

**Logs**:
   ```
serializers.serialize("json", get_objects(), indent=None, stream=out)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/core/serializers/__init__.py", line 129, in serialize
    s.serialize(queryset, **options)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/core/serializers/base.py", line 80, in serialize
    for count, obj in enumerate(queryset, start=1):
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/backends/base/creation.py", line 117, in get_objects
    for obj in queryset.iterator():
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 886, in execute_sql
    raise original_exception
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 876, in execute_sql
    cursor.execute(sql, params)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 328, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: main_chatsession
```

**Would you like to work on this issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
**Status**:
- [x] Ready
- [ ] In Development
- [ ] On Hold

**Which issue does this PR fix?**: 
  fixes #85

### CODE REVIEW

**Comment by tapasweni-pathak** [comment]
@vaibhavsingh97 It will be better to have these two commits in separate pull request. You are closing two issues with this pull request, isn't it? or you forgot to clean up? :)

**Comment by tapasweni-pathak** [comment]
@vaibhavsingh97 Let's fix this how Jarif has suggested it in #84. Plus this has two commits which fixes two different issues. Cleanup?

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ install:
 before_script:
   - psql -c 'create database travis_ci_test;' -U postgres
   - cd oshc/
+  - python manage.py makemigrations --noinput
```
Comment (tapaswenipathak) [inline .travis.yml:21]: @vaibhavsingh97 Let's fix this how Jarif has suggested it in #84. Plus this has two commits which fixes two different issues. Cleanup?

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (vaibhavsingh97) [reply .travis.yml:21]: @tapasweni-pathak Now we don't need this PR. I am closing as stated in #84

**Pull request closed without merging**